### PR TITLE
[Tools] CouchDB_Import_MRI.php cleanup

### PR DESCRIPTION
**Pull request by laemtl**
- Remove the side effect in CouchDB_MRI_Importer.php
- Migrate CouchDB_MRI_Importer.php in importers/
- Migrate the import statements in CouchDB_MRI_Importer.php

### CODE REVIEW

**Comment by laemtl** [comment]
@cmadjar Your pb should be fixed with #8369.

**Review by cmadjar — APPROVED** [review]
LGTM

File: tools/importers/CouchDB_MRI_Importer.php [hunk 1]
```diff
@@ -11,8 +11,11 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-require_once __DIR__ . "/../vendor/autoload.php";
-require_once 'generic_includes.php';
+require_once __DIR__ . '/../../vendor/autoload.php';
```
Comment (driusan) [inline tools/importers/CouchDB_MRI_Importer.php:14]: This is included by generic_includes

File: tools/importers/CouchDB_MRI_Importer.php [hunk 2]
```diff
@@ -11,8 +11,11 @@
...
+require_once __DIR__ . '/../generic_includes.php';
```
Comment (driusan) [inline tools/importers/CouchDB_MRI_Importer.php:15]: requiring is a side-effect, it should be in the other file not the class definition file.

File: tools/importers/CouchDB_MRI_Importer.php [hunk 3]
```diff
@@ -11,8 +11,11 @@
...
+require_once 'CouchDB.class.inc';
+require_once 'Database.class.inc';
+require_once 'FeedbackMRI.class.inc';
```
Comment (driusan) [inline tools/importers/CouchDB_MRI_Importer.php:18]: These are all autoloaded

**Review by driusan — CHANGES_REQUESTED** [review]

**Comment by laemtl** [comment]
@driusan Done

**Pull request merged**